src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -53,12 +54,16 @@ TEST_CASE("generate load in protocol 1")
 TEST_CASE("generate load with unique accounts", "[loadgen]")
 {
     Hash networkID = sha256(getTestConfig().NETWORK_PASSPHRASE);
+    uint32_t const nAccounts = 1000;
```

Comment:
I think this tests looks good, but it kinda canibalizes our only explicit create accounts load test. Should we break this off into a separate test to preserve the legacy account generation until it's deprecated (which I assume is the plan)?

Reply:
Ya that's probably good enough, especially since we want to get rid of the old account generation soon anyway.